Tiltfile.dev

```diff
@@ -1,6 +1,6 @@
 local_resource(
   name = 'dev:overseas-entities-web',
-  cmd = 'npm install && npm run build',
+  cmd = 'rm -rf ./dist && npm install && npm run build',
```

Comment:
Deleting the `dist` folder here ensures that the OE web Tilt service fails to start up if in development mode and the service fails to build, e.g. if node or npm versions in the UNIX shell don't match the required version.

Without it, the Tilt dev:overseas-entities-web 'task' fails but the overseas-entities-web one doesn't (and seems to use a potentially older version of the compiled code, in the `dist` folder).